lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -74,10 +74,14 @@ export default function AssignmentActivity() {
         {assignment.data && (
           <div className="mb-3 mt-1 w-full">
             <DashboardBreadcrumbs
+              allCoursesLink={urlWithFilters({ studentIds }, { path: '' })}
```

Comment:
The paths for different URLs are currently spread out in various places which makes it difficult to find and update all references. I think the frontend needs an equivalent of the backend's `request.route_url` function.